packages/@sanity/core/src/actions/graphql/gen2/filters/documentFilters.js

```diff
@@ -1,6 +1,6 @@
 function createDocumentFilters() {
   return {
-    name: 'DocumentFilter',
+    name: 'GlobalDocumentFilter',
```

Comment:
This would be a breaking change in the (likely rare?) case that this is being done:

```gql
query GetAllUser($docFilter: DocumentFilter) {
  allUser(_: $docFilter) {
    _id
  }
}
```